snapshotTestContent/DeletedTables_0.43/current_snapshots/snapshot_461_0.json

```diff
@@ -0,0 +1,5416 @@
+                      "contents": "{\"usedRoutes\":[],\"gcData\":{\"gcNodes\":{\"/root\":[\"/1d10a893-cacd-4888-acfa-fdfb8fa83b34/3acdfaa0-d288-4cb4-8e90-613b585cfcf9\",\"/1d10a893-cacd-4888-acfa-fdfb8fa83b34/a443af9a-210f-40f8-a64c-d89723bcdc2e\",\"/1d10a893-cacd-4888-acfa-fdfb8fa83b34\"],\"/a443af9a-210f-40f8-a64c-d89723bcdc2e\":[\"/1d10a893-cacd-4888-acfa-fdfb8fa83b34\"],\"/3acdfaa0-d288-4cb4-8e90-613b585cfcf9\":[\"/1d10a893-cacd-4888-acfa-fdfb8fa83b34\"],\"/\":[\"/1d10a893-cacd-4888-acfa-fdfb8fa83b34/root\",\"/1d10a893-cacd-4888-acfa-fdfb8fa83b34/3acdfaa0-d288-4cb4-8e90-613b585cfcf9\",\"/1d10a893-cacd-4888-acfa-fdfb8fa83b34/a443af9a-210f-40f8-a64c-d89723bcdc2e\"]}}}",
+                      "encoding": "utf-8"
+                    }
+                  }
+                ],
+                "unreferenced": true
+              },
+              "mode": "040000",
+              "type": "Tree"
+            },
+            {
+              "path": "25dd2996-f667-472f-8c6a-5f4859216e03",
+              "value": {
+                "entries": [
+                  {
+                    "path": ".channels",
+                    "value": {
+                      "entries": [
+                        {
+                          "path": "cf8168f5-ac44-46a4-98d4-3eb099b041ab",
+                          "value": {
+                            "entries": [
+                              {
+                                "path": ".attributes",
+                                "mode": "100644",
+                                "type": "Blob",
+                                "value": {
+                                  "contents": "{\"type\":\"https://graph.microsoft.com/types/mergeTree\",\"snapshotFormatVersion\":\"0.1\",\"packageVersion\":\"0.43.1\"}",
+                                  "encoding": "utf-8"
+                                }
+                              },
+                              {
+                                "path": "content",
+                                "value": {
+                                  "entries": [
+                                    {
+                                      "path": "header",
+                                      "mode": "100644",
+                                      "type": "Blob",
+                                      "value": {
+                                        "contents": "{\"chunkStartSegmentIndex\":0,\"chunkSegmentCount\":2,\"chunkLengthChars\":9,\"totalLengthChars\":9,\"totalSegmentCount\":2,\"chunkSequenceNumber\":461,\"segmentTexts\":[{\"text\":\"Row2Col1\",\"props\":{\"attribution\":{\"id\":\"naagar@microsoft.com\",\"name\":\"Navin Agarwal\",\"email\":\"naagar@microsoft.com\",\"timestamp\":1627344000000}}},{\"marker\":{\"refType\":1},\"props\":{\"markerId\":\"d44d9bef-5f69-402e-a6e3-10c4100d266f\",\"nodeType\":\"Paragraph\"}}],\"headerMetadata\":{\"orderedChunkMetadata\":[{\"id\":\"header\"}],\"sequenceNumber\":461,\"totalLength\":9,\"totalSegmentCount\":2}}",
```

Comment:
looks like this file was not sanitized, as it still contains (your) user info.  can you sanitize this with the sanitization tool

Reply:
@anthony-murphy Thanks for pointing this out. Sanitized the snapshot.